app/api/custom_system_roles.py

```diff
@@ -20,6 +22,18 @@ class CustomSystemRoleDetail(ResourceDetail):
     """
     Custom System Role detail by id
     """
+    def before_get_object(self, view_kwargs):
+        """
+        before get method for user object
+        :param view_kwargs:
+        :return:
+        """
+        if view_kwargs.get('role_id') is not None:
```

Comment:
usually writing `if if view_kwargs.get('role_id'):` is enough. :) 
Not worth requesting changes for but something to keep in mind.